src/blocks/ImageCard.scss

```diff
@@ -11,8 +11,14 @@
   --grid-gap-mobile: var(--bloom-s2);
   --grid-gap-desktop: var(--bloom-s4);
   --leader-width: var(--bloom-width-2-3rd);
+  --grid-template-columns-mobile: 1fr 1fr;
+  --grid-template-rows-mobile: 1.2fr 0.5fr;
+  --grid-template-areas-mobile:
+    "Main Main"
+    "Secondary-1 Secondary-2";
 
   position: relative;
+  margin-bottom: var(--grid-gap-desktop);
```

Comment:
I don't think we'd want to include this spacing here but rather include it on the core side as part of the implementation of this component. Additionally, we'd want to avoid using --grid-gap-desktop even though it is the intended size. 

Some context on the css variables and ui-components but apologies if you've already heard this: This library is used by multiple different sites and with slightly different implementations. The css variables allow the consumers of the library to customize specific parts of the component by overriding specific values without changing the general design of the component. For this use above, a consumer might override this variable to show a different gap in the multi-image grid but would then be surprised that whatever spacing they want forcibly changes the margin below the entire image card as well.

Reply:
+1, or as a separate variable!